litex_boards/platforms/alinx_ax7020.py

```diff
@@ -0,0 +1,112 @@
+        Subsignal("tx", Pins("W19"), IOStandard("LVCMOS33")),
+        Subsignal("rx", Pins("W18"), IOStandard("LVCMOS33")),
+    ),
+]
+
+_ps7_io = [
+    # PS7
+    ("ps7_clk",   0, Pins("E7")),
+    ("ps7_porb",  0, Pins(1)),
+    ("ps7_srstb", 0, Pins(1)),
+    ("ps7_mio",   0, Pins(54)),
+    ("ps7_ddram", 0,
+        Subsignal("addr",    Pins(15)),
+        Subsignal("ba",      Pins(3)),
+        Subsignal("cas_n",   Pins(1)),
+        Subsignal("ck_n",    Pins(1)),
+        Subsignal("ck_p",    Pins(1)),
+        Subsignal("cke",     Pins(1)),
+        Subsignal("cs_n",    Pins(1)),
+        Subsignal("dm",      Pins(4)),
+        Subsignal("dq",      Pins(32)),
+        Subsignal("dqs_n",   Pins(4)),
+        Subsignal("dqs_p",   Pins(4)),
+        Subsignal("odt",     Pins(1)),
+        Subsignal("ras_n",   Pins(1)),
+        Subsignal("reset_n", Pins(1)),
+        Subsignal("we_n",    Pins(1)),
+        Subsignal("vrn",     Pins(1)),
+        Subsignal("vrp",     Pins(1)),
+    ),
+]
+
+
+ps7_config_variants = {
+    "common" : {
+        "PCW_FPGA_FCLK0_ENABLE"         : "1",
+        "PCW_FPGA0_PERIPHERAL_FREQMHZ"  : "100",
+        "PCW_S_AXI_HP0_FREQMHZ"         : "100",
+        "PCW_S_AXI_HP0_DATA_WIDTH"      : "32",
+        #"PCW_UART1_BAUD_RATE"           : "115200",
+        #"PCW_EN_UART1"                  : "1",
+        #"PCW_UART1_PERIPHERAL_ENABLE"   : "1",
+        #"PCW_UART1_UART1_IO"            : "MIO 48 .. 49",
+        "PCW_PRESET_BANK1_VOLTAGE"      : "LVCMOS 1.8V",
+        #"PCW_USE_M_AXI_GP0"             : "1",
+        "PCW_USE_S_AXI_HP0"             : "1",
+        "PCW_USB0_PERIPHERAL_ENABLE"    : "1",
+        #"PCW_USB0_USB0_IO"              : "MIO 28 .. 39",
+        #"PCW_USB0_RESET_ENABLE"         : "1",
+        #"PCW_USB0_RESET_IO"             : "MIO 46",
+        #"PCW_EN_USB0"                   : "1",
+        "PCW_UIPARAM_DDR_PARTNO"        : "MT41K256M16 RE-125",
+    },
+}
+
+# Connectors ---------------------------------------------------------------------------------------
+
+_connectors = [
+    ("pmodb",    "B12 B12 C12"),
+    ("pmodhdmi", "N18 P19 V20 W20 T20 U20 N20 P20 R18 R16 Y18 Y19 V16"),
+    ("pmodj10",  "W19 W18 R14 P14 Y17 Y16 W15 V15 Y14 W14 P18 N17 U15 U14 P16 P15 U17 T16 V18 V17 T15 T14 V13 U13 W13 V12 U12 T12 T10 T11 A20 B19 B20 C20"),
+    ("pmodj11",  "F17 F16 F20 F19 G20 G19 H18 J18 L20 L19 M20 M19 K18 K17 J19 K19 H20 J20 L17 L16 M18 M17 D20 D19 E19 E18 G18 G17 H17 H16 G15 H15 J14 K14"),
+]
+
+# Platform -----------------------------------------------------------------------------------------
+
+class Platform(Xilinx7SeriesPlatform):
+    default_clk_name   = "clk50"
+    default_clk_freq   = 50e6
+    default_clk_period = 1e9/default_clk_freq
+
+    def __init__(self):
+        self.ps7_config = ps7_config_variants["common"]
+        Xilinx7SeriesPlatform.__init__(self, "xc7z020clg400-2", _io, _connectors, toolchain="vivado")
+        self.add_extension(_ps7_io)
```

Comment:
why not including _ps7_io in _io (as done for digilen_arty_z7 or digilent_zedboard)?